targets/unity-v2/source/ExampleTestProject/Assets/Testing/Tests/Client/WebRequestTests.cs

```diff
@@ -0,0 +1,95 @@
+using PlayFab.ClientModels;
+using PlayFab.Internal;
+
+namespace PlayFab.UUnit
+{
+    public class WebRequestTests : UUnitTestCase
+    {
+        private Action _tickAction = null;
+        private TestTitleDataLoader.TestTitleData testTitleData;
+        private static readonly PlayFabApiSettings clientSettings = new PlayFabApiSettings();
+        private static readonly PlayFabAuthenticationContext clientContext = new PlayFabAuthenticationContext();
+        
+        private static readonly PlayFabClientInstanceAPI clientInstance = new PlayFabClientInstanceAPI(clientSettings, clientContext);
+
+
+        int originalTimeout = 0;
+        WebRequestType originalWebRequestType = WebRequestType.UnityWebRequest;
+
+        public override void ClassSetUp()
+        {
+            testTitleData = TestTitleDataLoader.LoadTestTitleData();
+            clientSettings.TitleId = testTitleData.titleId;
+        }
+
+        public override void SetUp(UUnitTestContext testContext)
+        {
+            originalTimeout = PlayFabSettings.RequestTimeout;
+            originalWebRequestType = PlayFabSettings.RequestType;
+
+            var titleInfoSet = !string.IsNullOrEmpty(clientSettings.TitleId);
+            if (!titleInfoSet)
+                testContext.Skip(); // We cannot do client tests if the titleId is not given
+        }
+
+        public override void Tick(UUnitTestContext testContext)
+        {
+            if (_tickAction != null)
+                _tickAction();
+        }
+
+        public override void TearDown(UUnitTestContext testContext)
+        {
+            RestoreTimeoutSettings();
+            clientSettings.AdvertisingIdType = null;
+            clientSettings.AdvertisingIdValue = null;
+            _tickAction = null;
+        }
+
+        public override void ClassTearDown()
+        {
+            clientInstance.ForgetAllCredentials();
+        }
+
+[UUnitTest]
+        private void UnityWebRequestTimeOutIgnoredTest(UUnitTestContext testContext)
+        {
+            originalTimeout = PlayFabSettings.RequestTimeout;
+            originalWebRequestType = PlayFabSettings.RequestType;
+
+            PlayFabSettings.RequestTimeout = 1;
+            PlayFabSettings.RequestType = WebRequestType.UnityWebRequest;
+
+            var loginRequest = new LoginWithCustomIDRequest
+            {
+               CustomId = PlayFabSettings.BuildIdentifier,
+               CreateAccount = true,
+            };
+
+            clientInstance.LoginWithCustomID(loginRequest, LoginTimeoutIgnored, LoginTimeoutIgnoredError, testContext);
+        }
+
+        private void LoginTimeoutIgnored(LoginResult result)
+        {
+            ((UUnitTestContext)result.CustomData).EndTest(UUnitFinishState.PASSED, null);
+        }
+
+        private void LoginTimeoutIgnoredError(PlayFabError result)
+        {
+            ((UUnitTestContext)result.CustomData).Fail("Failed with unexpected error: " + result.GenerateErrorReport());
```

Comment:
Shouldn't this be the pass condition?

Reply:
Only if it was to check for timing out. But this test is the opposite. 

---
In reply to: [517627568](https://github.com/PlayFab/SDKGenerator/pull/735#discussion_r517627568) [](ancestors = 517627568)